src/raw_api.c

```diff
@@ -84,6 +84,7 @@ ffi_raw_to_ptrarray (ffi_cif *cif, ffi_raw *raw, void **args)
 	
 #if !FFI_NO_STRUCTS  
 	case FFI_TYPE_STRUCT:
+    case FFI_TYPE_EXT_VECTOR:
```

Comment:
This fails to build when FFI_TYPE_STRUCT == FFI_TYPE_EXT_VECTOR.

See the failing moxie-elf build here: https://travis-ci.org/libffi/libffi/jobs/353854020

../src/raw_api.c:87:5: error: duplicate case value
     case FFI_TYPE_EXT_VECTOR:
     ^~~~
../src/raw_api.c:86:2: note: previously used here
  case FFI_TYPE_STRUCT:
  ^~~~